library/src/main/java/com/anjlab/android/iab/v3/BillingProcessor.java

```diff
@@ -489,6 +490,42 @@ public boolean isOneTimePurchaseWithExtraParamsSupported(Bundle extraParams)
 		return isOneTimePurchaseExtraParamsSupported;
 	}
 
+	/**
+	 * Checks does API support version 6 required to request purchase history
```

Comment:
lets change this descritons to `Checks if API supports version 6 which required to request purchase history`.

 I ❤️ the descriptions in this PR BTW

Reply:
Updated